Dockerfile

```
@@ -0,0 +1,4 @@
+FROM golang:1.15
+
+RUN apt-get update
+RUN apt-get -y install zip
```

best to run as one command to reduce layers, and also clean up afterwards:

```suggestion
RUN apt-get update && apt-get -y install zip && rm -rf /var/lib/apt/lists/*
```